modules/interactive_music/doc_classes/AudioStreamInteractive.xml

```diff
@@ -0,0 +1,230 @@
+			<description>
+				Return the list of transitions (from, to interleaved).
+			</description>
+		</method>
+		<method name="get_transition_to_time" qualifiers="const">
+			<return type="int" enum="AudioStreamInteractive.TransitionToTime" />
+			<param index="0" name="from_clip" type="int" />
+			<param index="1" name="to_clip" type="int" />
+			<description>
+				Return the destination time position for a transition (see [method add_transition]).
+			</description>
+		</method>
+		<method name="has_transition" qualifiers="const">
+			<return type="bool" />
+			<param index="0" name="from_clip" type="int" />
+			<param index="1" name="to_clip" type="int" />
+			<description>
+				Return true if a given transition exists (was added via [method add_transition]).
+			</description>
+		</method>
+		<method name="is_transition_holding_previous" qualifiers="const">
+			<return type="bool" />
+			<param index="0" name="from_clip" type="int" />
+			<param index="1" name="to_clip" type="int" />
+			<description>
+				Return whether a transition uses the [i]hold previous[/i] functionality (see [method add_transition]).
+			</description>
+		</method>
+		<method name="is_transition_using_filler_clip" qualifiers="const">
+			<return type="bool" />
+			<param index="0" name="from_clip" type="int" />
+			<param index="1" name="to_clip" type="int" />
+			<description>
+				Return whether a transition uses the [i]filler clip[/i] functionality (see [method add_transition]).
+			</description>
+		</method>
+		<method name="set_clip_auto_advance">
+			<return type="void" />
+			<param index="0" name="clip_index" type="int" />
+			<param index="1" name="mode" type="int" enum="AudioStreamInteractive.AutoAdvanceMode" />
+			<description>
+				Set whether a clip will auto-advance by changing the auto-advance mode.
+			</description>
+		</method>
+		<method name="set_clip_auto_advance_next_clip">
+			<return type="void" />
+			<param index="0" name="clip_index" type="int" />
+			<param index="1" name="auto_advance_next_clip" type="int" />
+			<description>
+				Set the index of the next clip towards which this clip will auto advance to when finished. If the clip being played loops, then auto-advance will be ignored.
+			</description>
+		</method>
+		<method name="set_clip_name">
+			<return type="void" />
+			<param index="0" name="clip_index" type="int" />
+			<param index="1" name="name" type="StringName" />
+			<description>
+				Set the name of the current clip (for easier identification).
+			</description>
+		</method>
+		<method name="set_clip_stream">
+			<return type="void" />
+			<param index="0" name="clip_index" type="int" />
+			<param index="1" name="stream" type="AudioStream" />
+			<description>
+				Set the [AudioStream] associated with the current clip.
+			</description>
+		</method>
+	</methods>
+	<members>
+		<member name="_transitions" type="Dictionary" setter="_set_transitions" getter="_get_transitions" default="{}">
```

Comment:
Looks like something that shouldn't be exposed. You should use PROPERTY_USAGE_INTERNAL for it.